### PR TITLE
DRB Stream Layer

### PR DESCRIPTION
**Pull request by lewfish**
This adds a layer for the streams (as predicted by TauDem) in the DRB, and is only selectable when the DRB is visible.

screen shot 2016-03-18 at 5 30 00 pm 2

##### Testing
* Copy this branch's version of `setupdb.sh` to a place accessible to psql (the app VM is good)
* Run `./setupdb.sh -d` to load data
* The DRB streams layer should only be selectable when the map viewport overlaps with the DRB.

Connects #1195

### CODE REVIEW

File: src/mmw/js/src/core/utils.js [hunk 1]
```diff
@@ -42,6 +43,43 @@ var utils = {
...
+                viewportPolygon = {
+                    type: 'Feature',
+                    geometry: {
+                        type: 'Polygon',
+                        coordinates: [[
+                            [mapBounds.getNorthEast().lng, mapBounds.getNorthEast().lat],
+                            [mapBounds.getSouthEast().lng, mapBounds.getSouthEast().lat],
+                            [mapBounds.getSouthWest().lng, mapBounds.getSouthWest().lat],
+                            [mapBounds.getNorthWest().lng, mapBounds.getNorthWest().lat],
+                            [mapBounds.getNorthEast().lng, mapBounds.getNorthEast().lat]
```
Comment (kdeloach) [inline src/mmw/js/src/core/utils.js]: Does the polygon not self-close if you omit the final point?

**Comment by kdeloach** [comment]
This is really cool. :+1:

File: src/mmw/js/src/core/utils.js (hunk 1 above)
Comment (lewfish) [inline src/mmw/js/src/core/utils.js]: Apparently turf-intersect doesn't work if you omit that final point.

**Comment by mmcfarland** [comment]
Looks good.  The network is so detailed, I think we may actually want to clamp down at `'minZoom': 11` in layer_settings until we can get the caching turned on.  

What's the total row count meant to be, 410,607?

File: src/mmw/mmw/settings/layer_settings.py [hunk 2]
```diff
@@ -119,6 +128,16 @@
         'minZoom': 10
     },
     {
+        'code': 'drb_streams',
+        'display': 'DRB Stream Network',
+        'table_name': 'drb_streams_50',
+        'boundary': False,
+        'stream': True,
+        'overlay': True,
+        'perimeter': drb_perimeter # Layer is only selectable when viewport
```
Comment (mmcfarland) [inline src/mmw/mmw/settings/layer_settings.py]: Sorry, with more context.  I think we should add a `'minZoom': 11` to this block.

**Comment by lewfish** [comment]
> Looks good. The network is so detailed, I think we may actually want to clamp down at 'minZoom': 11 in layer_settings until we can get the caching turned on.
> 

Will do.

> What's the total row count meant to be, 410,607?

I have 586,285 rows. I ran `SELECT COUNT(*) FROM drb_streams_50;`

**Comment by mmcfarland** [comment]
I had the old DRB streams table and had the count off of that, 586,285 confirms.

File: src/mmw/mmw/settings/production.py [hunk 3]
```diff
@@ -105,7 +105,6 @@
     'LayerAttribution',
     'LayerSelector',
     'LocateMeButton',
-    # 'StreamControl',
```
Comment (mmcfarland) [inline src/mmw/mmw/settings/production.py]: There is a similar configuration still in `settings/base.py` and `core/tests.js`

**Comment by kdeloach** [comment]
+1 I see streams.

**Pull request merged**